clients/lsp-sorbet.el

```diff
@@ -0,0 +1,56 @@
+;;; lsp-sorbet.el --- Sorbet server configuration  -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2020
+
+;; Author: Christopher Wilson <chris@sencjw.com>
+;; Keywords:
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; lsp-sorbet client
+
+;;; Code:
+
+(require 'lsp-mode)
+
+(defgroup lsp-sorbet nil
+  "LSP support for Ruby, using the Sorbet language server."
+  :group 'lsp-mode
+  :link '(url-link "https://github.com/sorbet/sorbet")
+  :package-version '(lsp-mode . "6.1"))
+
+(defcustom lsp-sorbet-use-bundler nil
+  "Run sorbet under bundler"
+  :type 'boolean
+  :group 'lsp-sorbet
+  :package-version '(lsp-mode . "6.1"))
+
+(defun lsp-sorbet--build-command ()
+  "Build sorbet command"
+  (let ((lsp-command '("srb" "typecheck" "--lsp" "--disable-watchman")))
+    (if lsp-sorbet-use-bundler
+              (append '("bundle" "exec") lsp-command)
+            lsp-command)))
+
+(lsp-register-client
+ (make-lsp-client
+  :new-connection (lsp-stdio-connection
```

Comment:
you should pass :priority as well. The priority will determine which server will kick in in case there are 2 servers present on the path.

Reply:
I guess the priority should be -2 I guess to make this with lower priority than the solargraph one.